docker/entrypoint.sh

```diff
@@ -30,6 +30,23 @@ install_local_deps() {
     done
 }
 
+process_init_files() {
+    local f
+    for f; do
+        case "$f" in
+            *.sh)
+                if [ -x "$f" ]; then
+                    echo "$0: running $f"
```

Comment:
please use `log_message()` function for consistency with the rest of the script.